packages/core/src/enums/FrameStatus.ts

```diff
@@ -0,0 +1,12 @@
+/** Status of a frame as it gets loaded */
+enum FrameStatus {
+  // Replicate is a duplicated image, from some larger distance
+  REPLICATE = 1,
+  // Nearby replicate is a duplicated image of a nearby image
+  NEARBY_REPLICATE = 2,
+  LOADING = 3,
+  LOSSY = 4,
+  DONE = 5,
+}
```

Comment:
Could you please provide the documentation for those who don't have it? Is it reasonable to rename REPLICATE as FAR_REPLICATE based on the documentation I'm reading?

Reply:
And maybe we call this ImageStatus? since you are using it in ICachedImage? or even maybe CachedImageStatus?

Reply:
Not CachedImageStatus as the images may well not be cached.  Calling it ImageStatus for now, although I preferred FrameStatus as that is the normal DICOM naming, whereas images there are both single and multi frame.